packages/elements/src/list/index.ts

```diff
@@ -353,22 +342,21 @@ export class List<T extends DataItem = ItemData> extends ControlElement {
    */
   protected highlightItem (item?: T, scrollToItem = false): void {
     if (item) {
-      const elementToFocus = this.elementFromItem(item);
-      const focus = this.activeElement && this.activeElement !== elementToFocus;
       this.clearHighlighted();
       this.composer.setItemPropertyValue(item, 'highlighted', true);
-      focus && elementToFocus?.focus({ preventScroll: true });
+      (this.tabIndex >= 0 && this.renderer.key && item.value) && this.setAttribute('aria-activedescendant', `${this.renderer.key}-${item.value}`);
       scrollToItem && this.scrollToItem(item);
     }
   }
 
-
   /**
    * Gets the available tabbable elements
    */
   protected get tabbableItems (): HTMLElement[] {
-    return Array.from(this.children)
-    .filter((el): el is HTMLElement => (el as HTMLElement).tabIndex >= 0);
+    return Array.from(this.children).filter((el): el is HTMLElement => {
+      const role = (el as HTMLElement).getAttribute('role');
+      return (role === 'option' || role === 'treeitem') && (el as HTMLElement).getAttribute('disabled') === null;
+    });
```

Comment:
Make it a bit more readable
```suggestion
    return Array.from(this.children).filter((el): el is HTMLElement => {
      if (el instanceof HTMLElement) {
        const isEnabled = !el.hasAttribute('disabled');
        const isOption = ['option', 'treeitem'].includes( el.getAttribute('role'));
        return isOption && isEnabled;
      }
      return false;
    });
```